README.md

```diff
@@ -0,0 +1,607 @@
+# Tariff Interpreter
+
+___
+This Interpreter is a reverence implementation for computing a JSON formatted bike-sharing tariff model.
+
+## Contents
```

Comment:
I'd like to have the following headings beforehand:
- Features
- Installation
- Usage

Not perfect, but as idea: https://github.com/nextbike/phrase-swift